src/encoded/types/target.py

```diff
@@ -21,7 +21,10 @@ class Target(Item):
 
     item_type = 'target'
     schema = load_schema('encoded:schemas/target.json')
-    embedded_list = Item.embedded_list + lab_award_attribution_embed_list + ['targeted_genome_regions.*']
+    embedded_list = Item.embedded_list + lab_award_attribution_embed_list + [
+        # GenomicRegion linkTo (is * really necessary?) - Will
+        'targeted_genome_regions.*'
+    ]
```

Comment:
Target was deprecated and replaced by Biofeature, so we might be  able to remove this embed or comment out the  calc prop altogether.

Reply:
Yes it can be removed.  I have it on my continuing growing list of things to do to minimize Target by removing any calcprops, inserts, etc that can be since it is deprecated - not sure when I'll get to it so if someone wants to take it on feel free.

Reply:
I've scrubbed the type aside from it's declaration, in case there is any metadata laying around. Feel free to "finalize" the removal at another time.